### PR TITLE
Remove deprecated Buildable.Owner field.

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Comment by Mailaender** [comment]
As discussed in IRC. Add an upgrade rule that just removes the removed parameters and maybe leave a note in the command line.

**Comment by pchote** [comment]
Updated.

**Comment by pchote** [comment]
I still need to add a rule to remove Owner:

**Comment by pchote** [comment]
Updated properly this time.

**Comment by Mailaender** [comment]
Needs a rebase.

**Comment by pchote** [comment]
Rebased.

File: OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs [hunk 1]
```diff
@@ -155,14 +152,9 @@ void CacheProduceables(Actor playerActor)
 			{
 				var bi = a.Traits.Get<BuildableInfo>();
 
-				// Can our race build this by satisfying normal prerequisites?
-				var buildable = !Info.RequireOwner || bi.Owner.Contains(Race);
-
 				// Checks if Prerequisites want to hide the Actor from buildQueue if they are false
```
Comment (penev92) [inline OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs]: There is no checking done here any more, so this comment should probably go as well.

File: OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs [hunk 2]
```diff
@@ -872,6 +872,23 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
...
+						node.Value.Nodes.RemoveAll(n => n.Key == "RequireOwner");
+
+					if (node.Key == "Buildable")
+					{
+						var removed = node.Value.Nodes.RemoveAll(n => n.Key == "Owner");
+						if (removed > 0)
+						{
+							Console.WriteLine("The 'Owner' field has been removed.");
+							Console.WriteLine("Please use prerequisites instead.");
+							Console.WriteLine("");
```
Comment (penev92) [inline OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs]: Minor style nit - an empty string is not needed here.
The method has an override that takes no arguments and just prints an empty line.

**Comment by penev92** [comment]
Cound't find anything broken ingame, so :+1:

**Comment by pchote** [comment]
Rebased.

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/46f038abc2fc643738b0f37a44fdb65e230fa0b4...57946714f328a79399237f02903226c4b221c7c2)

**Pull request merged**